community/good-member/README.md

```diff
@@ -0,0 +1,133 @@
+# Being a Good Exercism Community Member
+
+We want Exercism to be a great place to spend time, whoever you are.
+To ensure that Exercism is that, we place a strong focus on being good community members.
+
+This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**
+
+There are a few principles to try and keep in mind as you contribute to Exercism.
+Whether you're mentoring, learning, hanging out in a chat space, or helping building things on GitHub, if you follow these ideas, you'll find that Exercism can be an incredibly fun and rewarding place to spend time.
+
+- Presume good intentions
+- Being wrong is awesome
+- You don't know what you don't know
+- Disagree constructively
+- Choose your words carefully
+
+## Presume good intentions
+
+The vast majority of people who are involved in Exercism do so because they want to help others.
+They're probably passionate about tech, maybe interested in education, but overwhelmingly want to be in a space where they're building something positive with others.
+
+That means that the majority of discussions are happening between two people who want the best for each and also for Exercism.
+
+Sometimes, when disagreements happen its easy to jump to the conclusion that the other person is being difficult, or not listening, or not understanding your point.
+It's easy to imagine they're sitting there feeling smug, polishing their ego, while they rile you up with their wrong opinions.
+In reality, they're much more likely to be trying to communicate something that they feel is important and not finding a way to do that in a way that is clear to others.
+
+When we step back and presume each other are **tying** to be kind, and trying to do the right things, and trying to be clear, it makes it a lot easier for us to loosen the grip on our own egos and **actively** try to listen to the other person's point.
+
+Which brings us on to...
+
+## Being wrong is awesome
+
+Learning that you're wrong about something is the best thing that can happen to you, because you're now less wrong than you were before.
```

Comment:
```suggestion
Learning that you're wrong about something is the _best thing that can happen to you_, because you're now **less wrong** than you were before.
```